gradle.properties

```diff
@@ -12,3 +12,4 @@ edcDeveloperEmail=markus.spiekermann@isst.fraunhofer.de
 edcScmConnection=scm:git:git@github.com:eclipse-edc/Connector.git
 edcWebsiteUrl=https://github.com/eclipse-edc/Connector.git
 edcScmUrl=https://github.com/eclipse-edc/Connector.git
+org.gradle.jvmargs=-Xmx4096m
```

Comment:
please remove custom jvm parameters, you can set them on your local gradle settings file `~/.gradle/gradle.properties`